hack/local-up-cluster.sh

```diff
@@ -401,6 +401,14 @@ contexts:
     name: local-up-cluster
 current-context: local-up-cluster
 EOF
+
+    # flatten the kubeconfig files to make them self contained
```

Comment:
no need for the temporary file. Just pipe the HERE document above through `kubectl config view --kubeconfig /dev/stdin --minify --flatten`